projects/plugins/jetpack/extensions/blocks/one-payment/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { InnerBlocks } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import icon from './icon';
+import variations from './variations';
+import { getIconColor } from '../../shared/block-icons';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'one-payment';
+export const title = __( 'One payment', 'jetpack' );
+export const settings = {
+	title,
+	description: (
+		<Fragment>
+			<p>{ __( 'One Payment', 'jetpack' ) }</p>
+			<p>{ __( 'Take payments on your website', 'jetpack' ) }</p>
+		</Fragment>
+	),
+	icon: {
+		src: icon,
+		foreground: getIconColor(),
+	},
+	category: 'earn',
+	keywords: [],
+	supports: {
```

Comment:
Support  for all of these properties could be added here at the root level, as an easy way of bringing them to all blocks. Given that the blocks are already out there though, I think it makes sense to add support to those individual blocks where each feature makes sense.

Reply:
I agree.
Given that this block is fundamentally just a container, we would only really need the bare minimum to make it work appropriately. Maybe just alignment and background color?

Reply:
It's a container of maximum one block. The blocks that can be the one block still exist and are listed in the inserter independently. 

What's the reason for supporting alignment and background colour at the container level? 

For context recurring-payments supports align already and @rcrdortiz I think is adding it to donations in #22470